src/containers/ProfileDetails/components/constants.js

```diff
@@ -0,0 +1,335 @@
+    probeResponseThresholdDb: -90,
+    clientDisconnectThresholdDb: -90,
+    eirpTxPower: 18,
+    bestApSettings: {
+      model_type: 'RadioBestApSettings',
+      mlComputed: true,
+      dropInSnrPercentage: 20,
+      minLoadFactor: 50,
+    },
+  },
+  is5GHzU: {
+    model_type: 'RfElementConfiguration',
+    radioType: 'is5GHzU',
+    radioMode: 'modeAC',
+    rf: 'TipWlan-rf',
+    beaconInterval: 100,
+    forceScanDuringVoice: 'disabled',
+    rtsCtsThreshold: 65535,
+    channelBandwidth: 'is80MHz',
+    mimoMode: 'twoByTwo',
+    maxNumClients: 100,
+    autoChannelSelection: false,
+    activeScanSettings: {
+      model_type: 'ActiveScanSettings',
+      enabled: true,
+      scanFrequencySeconds: 10,
+      scanDurationMillis: 65,
+    },
+    neighbouringListApConfig: {
+      model_type: 'NeighbouringAPListConfiguration',
+      minSignal: -85,
+      maxAps: 25,
+    },
+    minAutoCellSize: -65,
+    perimeterDetectionEnabled: true,
+    channelHopSettings: {
+      model_type: 'ChannelHopSettings',
+      noiseFloorThresholdInDB: -75,
+      noiseFloorThresholdTimeInSeconds: 180,
+      nonWifiThresholdInPercentage: 50,
+      nonWifiThresholdTimeInSeconds: 180,
+      obssHopMode: 'NON_WIFI',
+    },
+    bestApEnabled: null,
+    multicastRate: 'auto',
+    managementRate: 'auto',
+    rxCellSizeDb: -90,
+    probeResponseThresholdDb: -90,
+    clientDisconnectThresholdDb: -90,
+    eirpTxPower: 18,
+    bestApSettings: {
+      model_type: 'RadioBestApSettings',
+      mlComputed: true,
+      dropInSnrPercentage: 30,
+      minLoadFactor: 40,
+    },
+  },
+  is5GHzL: {
+    model_type: 'RfElementConfiguration',
+    radioType: 'is5GHzL',
+    radioMode: 'modeAC',
+    rf: 'TipWlan-rf',
+    beaconInterval: 100,
+    forceScanDuringVoice: 'disabled',
+    rtsCtsThreshold: 65535,
+    channelBandwidth: 'is80MHz',
+    mimoMode: 'twoByTwo',
+    maxNumClients: 100,
+    autoChannelSelection: false,
+    activeScanSettings: {
+      model_type: 'ActiveScanSettings',
+      enabled: true,
+      scanFrequencySeconds: 10,
+      scanDurationMillis: 65,
+    },
+    neighbouringListApConfig: {
+      model_type: 'NeighbouringAPListConfiguration',
+      minSignal: -85,
+      maxAps: 25,
+    },
+    minAutoCellSize: -65,
+    perimeterDetectionEnabled: true,
+    channelHopSettings: {
+      model_type: 'ChannelHopSettings',
+      noiseFloorThresholdInDB: -75,
+      noiseFloorThresholdTimeInSeconds: 180,
+      nonWifiThresholdInPercentage: 50,
+      nonWifiThresholdTimeInSeconds: 180,
+      obssHopMode: 'NON_WIFI',
+    },
+    bestApEnabled: null,
+    multicastRate: 'auto',
+    managementRate: 'auto',
+    rxCellSizeDb: -90,
+    probeResponseThresholdDb: -90,
+    clientDisconnectThresholdDb: -90,
+    eirpTxPower: 18,
+    bestApSettings: {
+      model_type: 'RadioBestApSettings',
+      mlComputed: true,
+      dropInSnrPercentage: 30,
+      minLoadFactor: 40,
+    },
+  },
+};
+
+const defaultSsidProfile = {
```

Comment:
remove everything but the details, and make the details the root. i.e. `defaultSsidProfile. appliedRadios`